### PR TITLE
chore(client): remove holiday donation banner

### PR DESCRIPTION
**Pull request by ahmaxed**
This pr removes our holiday donation banner from learn.

Checklist:

- [x] I have read [freeCodeCamp's contribution guidelines](https://contribute.freecodecamp.org).
- [x] My pull request has a descriptive title (not a vague title like `Update index.md`)
- [x] My pull request targets the `main` branch of freeCodeCamp.
- [x] I have tested these changes either locally on my machine, or GitPod.

Closes #XXXXX

### CODE REVIEW

File: client/src/components/Intro/index.tsx [hunk 1]
```diff
@@ -70,24 +67,6 @@ const Intro = ({
         ) : (
           ''
         )}
-        <Alert bsStyle='info' className='annual-donation-alert'>
-          <p>
-            <b>{t('learn.season-greetings-fcc')}</b>
-          </p>
-          <p>{t('learn.if-getting-value')}</p>
```
Comment (nhcarrigan) [inline client/src/components/Intro/index.tsx:77]: A quick note that we should probably remove these keys from the English JSON too

**Review by ojeytonwilliams — COMMENTED** [review]
LGTM :+1:, but let's get rid of the redundant keys Nick mentioned.

File: client/src/pages/learn.tsx [hunk 2]
```diff
@@ -114,7 +96,7 @@ function LearnPage({
 
 LearnPage.displayName = 'LearnPage';
 
-export default connect(mapStateToProps, mapDispatchToProps)(LearnPage);
+export default connect(mapStateToProps, null)(LearnPage);
```
Comment (ojeytonwilliams) [inline client/src/pages/learn.tsx:99]:
```suggestion
export default connect(mapStateToProps)(LearnPage);
```
Nitpick.

**Review by ojeytonwilliams — APPROVED** [review]
LGTM.  One nitpick, but I don't want to block merging.

**Review by ShaunSHamilton — APPROVED** [review]
LGTM 👍

File: client/src/pages/learn.tsx (hunk 2 above)
Comment (ShaunSHamilton) [reply client/src/pages/learn.tsx:99]: Resolving to merge - Ignoring Oliver.

**Pull request merged**